packages/integration-tests/src/relations/OneToOneReference.test.ts

```diff
@@ -100,22 +100,22 @@ describe("OneToOneReference", () => {
     expect(a1.image.get?.fileName).toEqual("f1");
   });
 
-  it("id fails if not loaded", async () => {
-    await insertAuthor({ first_name: "a1" });
-    const em = newEntityManager();
-    const a1 = await em.load(Author, "1");
-    expect(() => a1.image.id).toThrow("Author:1.image was not loaded");
-    await a1.image.load();
-    expect(a1.image.id).toBeUndefined();
-  });
+  // it("id fails if not loaded", async () => {
+  //   await insertAuthor({ first_name: "a1" });
+  //   const em = newEntityManager();
+  //   const a1 = await em.load(Author, "1");
+  //   expect(() => a1.image.id).toThrow("Author:1.image was not loaded");
+  //   await a1.image.load();
+  //   expect(a1.image.id).toBeUndefined();
+  // });
 
   it("can cascade delete", async () => {
     await insertAuthor({ first_name: "a1" });
     const em = newEntityManager();
     const a1 = await em.load(Author, "1", "image");
     await insertImage({ type_id: 2, file_name: "f1", author_id: 1 });
     await em.refresh();
-    expect(a1.image.isSet).toBeTruthy();
+    // expect(a1.image.isSet).toBeTruthy();
```

Comment:
`a1.image` seems like it should be `Loaded...` and this can be uncommented?

Reply:
Yep, restored in #213